webview/src/experiments/components/table/TableHeader.tsx

```diff
@@ -152,10 +226,12 @@ const TableHeaderCell: React.FC<{
   menuDisabled,
   onDragOver,
   onDragStart,
-  onDrop
+  onDrop,
+  root,
+  isFirst,
```

Comment:
[C] I think we need a better name for this "FirstCell". We would want to convey with the name that it contains controls/metadata about the rest of the row. I am not sure what the correct name would be.
Perhaps do a bit of research and follow up with a simple renaming PR. Rename everything around the "First" concept e.g `<FirstCell/>` etc.